### PR TITLE
fix: Invalid `Image` should not display

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

When `Image` source is invalid, it does display "broken image" icon on WASM. If empty, it shows border.

image

## What is the new behavior?

When `Image` source is invalid, it does not display at all.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
Nice!

**Review by Xiaoy312 — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/Image/HtmlImage.wasm.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿#nullable enable
+
+namespace Windows.UI.Xaml.Controls;
+
+public class HtmlImage : UIElement
+{
+	public HtmlImage() : base("img")
+	{
+		// Avoid the "drag effect" which is set by default in browsers
+		SetAttribute("draggable", "false");
```
Comment (carldebilly) [inline src/Uno.UI/UI/Xaml/Controls/Image/HtmlImage.wasm.cs:10]: Some browsers may also require css for this like  `user-drag: none` and the prefixed versions.

https://caniuse.com/?search=user-drag

**Review by carldebilly — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/Image/HtmlImage.wasm.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Controls/Image/HtmlImage.wasm.cs:10]: Looks like we should set it by default in css?

**Comment by MartinZikmund** [comment]
@carldebilly @jeromelaban Ok, so the actual reason was a bit different - in fact the `GetPhysicalRect` returns a rect which is evaluated when accessed. So even though I queried it before the panel was hidden, it was evaluated only after - and when visibility is `Collapsed`, the element is hidden and its `left` and `top` in CSS are shifted to -10000 - which is what the rect then returns 😀

**Comment by jeromelaban** [comment]
That is a tricky one indeed, I don't think can change much for that, considering that even in other frameworks, invisible elements don't even provide the element to start with (for android I'm sure of, at least).

**Pull request merged**